src/cliActions.js

```diff
@@ -119,13 +119,12 @@ const signingKeyChangePW = (
   });
 
 /**
- * @desc Streamming Dapp
+ * @desc Check existing of signing key, and decrypt the key
```

Comment:
`Check existing of signing key, and decrypt the key` sorry, what do you mean? I guess it should be `check if signing key exist and decrypt it`